fastplotlib/widgets/image.py

```diff
@@ -262,31 +263,19 @@ def __init__(
         data: Union[np.ndarray, List[np.ndarray]
             array-like or a list of array-like
 
-        dims_order: Optional[Union[str, Dict[np.ndarray, str]]]
-            | ``str`` or a dict mapping to indicate dimension order
-            | a single ``str`` if ``data`` is a single array, or a list of arrays with the same dimension order
-            | examples: ``"xyt"``, ``"tzxy"``
-            | ``dict`` mapping of ``{array_index: axis_order}`` if specific arrays have a non-default axes order.
-            | "array_index" is the position of the corresponding array in the data list.
-            | examples: ``{array_index: "tzxy", another_array_index: "xytz"}``
-
-        slider_dims: Optional[Union[str, int, List[Union[str, int]]]]
-            | The dimensions for which to create a slider
-            | can be a single ``str`` such as **"t"**, **"z"** or a numerical ``int`` that indexes the desired dimension
-            | can also be a list of ``str`` or ``int`` if multiple sliders are desired for multiple dimensions
-            | examples: ``"t"``, ``["t", "z"]``
-
         window_funcs: Dict[Union[int, str], int]
-            | average one or more dimensions using a given window
-            | if a slider exists for only one dimension this can be an ``int``.
-            | if multiple sliders exist, then it must be a `dict`` mapping in the form of: ``{dimension: window_size}``
-            | dimension/axes can be specified using ``str`` such as "t", "z" etc. or ``int`` that indexes the dimension
-            | if window_size is not an odd number, adds 1
-            | use ``None`` to disable averaging for a dimension, example: ``{"t": 5, "z": None}``
+            | You may want to apply functions along certain axes of your data before displaying.
+            | This parameter allows you pass in a dictionary; each key is an axis (allowed axes: "t" and "z").
+            | The value is a tuple: (Callable, window_size). Every callable must take "axis" as an input.
+            | A typical signature might be def my_func(img: np.ndarray, axis=0) -> np.ndarray:
+            | To use window_funcs, you do something like: window_funcs = {"t": (t_callable, window_size_t),
+            | "z": (z_callable, window_size_z)}. So at time index "50", if window_size_t is 10, we look ed at frames
+            | (50 - 5, 50 + 5), etc.
 
         frame_apply: Union[callable, Dict[int, callable]]
-            | apply a function to slices of the array before displaying the frame
-            | pass a single function or a dict of functions to apply to each array individually
+            | apply a function to frames of data along the "t" and "z" axes before displaying a single image
+            | (think time-averaging in local windows, etc.).
+            | Pass a single function or a dict of functions to apply to each array individually
             | examples: ``{array_index: to_grayscale}``, ``{0: to_grayscale, 2: threshold_img}``
```

Comment:
spatial filters is another good example, various kernels

Reply:
@kushalkolar spatial filters is a good example for frame_apply not window_function right?

Reply:
yup!